.github/workflows/check-pr-labels.yml

```diff
@@ -0,0 +1,37 @@
+name: "Check PR has required labels"
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - opened
+      - synchronize
+      - labeled
+      - unlabeled
+
+jobs:
+  check-pr-labels:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+            fetch-depth: 0 # This is needed to checkout all branches
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: yarn
+
+      - name: Install dependencies
+        run: yarn --immutable
```

Comment:
maybe i'm missing something here, do we need to install for the dependencies for label check? can we just have `@actions` ones?
```
import * as core from '@actions/core';
import { context, getOctokit } from '@actions/github';
import { GitHub } from '@actions/github/lib/utils';
```

Reply:
Ah i see i see, i thought we were installing all the dependencies. this is clear now ~ 🙏🏻